app-development-wg/README.md

```diff
@@ -13,8 +13,8 @@ or the meetings described below.
 
 ## Meetings
 
-* Meeting schedule: 1st and 3rd Wednesday of each month at [1600 UTC](https://www.timeanddate.com/worldclock/converter.html?iso=20221213T160000&p1=1440)
-   
+* Meeting schedule: Every two weeks on Wednesday @ 1600 Europe/London at [1600 Europe/London](https://www.timeanddate.com/worldclock/converter.html?iso=20221213T160000&p1=136)

```

Comment:
in [1600 Europe/London], does that infer BST or GMT? (how should daylight savings be handled)

Reply:
"Europe/London" implies local time (inc DST), but not everyone will know that.

How about we change this to "Every two weeks on Wednesday @ 4pm Europe/London local time" ?

Reply:
updated code to reflect that use "local time" - hopefully removing confusion over GMT/BST